cleanup: GEP-1324 Typo - Fix indentation (#1724)
kubernetes-sigs/gateway-api: site-src/geps/gep-1324.md

    1. I want to enforce that all traffic within my cluster is encrypted.
    2. I want to have strict isolation and control at namespace level, so a bug/malicious user can't impact other namespaces
    3. I want to be able to allow app owners to gradually opt-in to a mesh (no mesh, L4 only, L7 enabled) so they can choose the right fit for their applications’ performance and compatibility goals.

>     4. Since mesh can be multi-tenant and hosting multiple services (e.g. foo or bar), as a mesh administrator I need to make sure a client can discover different services. Here are few possible ways:

Reviewer: ```suggestion
    4. Since mesh can be multi-tenant and hosting multiple services (e.g. foo or bar), as a mesh administrator I need to make sure a client can discover different services. Here are a few possible ways:
```

        1. Each service is allocated a unique IP and port

>         2. Or Each service must use unique Host name

Reviewer: ```suggestion
        2. Or Each service must use a unique hostname
```

>         3. Or a unique port and protocol,  (80:http, 443 tls)

Reviewer: ```suggestion
        3. Or a unique port and protocol (80:http, 443:tls)
```
